src/services/compare-appian/handlers/sendReport.ts

```diff
@@ -0,0 +1,76 @@
+import {
+  sendAttachment,
+  trackError,
+  scanTable,
+  getCsvFromJson,
+} from "../../../libs";
+
+interface Data {
+  id: string;
+  appianSubmitted: boolean;
+  SPA_ID: string;
+  iterations: number;
+  secSinceAppianSubmitted: string;
+  appianSubmittedDate: string;
+  seatoolExist: boolean;
+  seatoolSubmissionDate?: string;
+  match?: boolean;
+}
+
+//! This work/logic will be done in another ticket
```

Comment:
perfect. yeah i thin k we want to have a separate status table for this comparison. we should also make the one that exists mmdl specific in its naming conventions. We can do that together, it shouldnt take more than an hour.